GAE/war/WEB-INF/webapp-security.xml

```diff
@@ -24,13 +24,6 @@
       <b:constructor-arg ref="oidcConfig"/>
    </b:bean>
 
-   <http pattern="/altlogin" use-expressions="true" entry-point-ref="alternativeEntryPoint" authentication-manager-ref="authenticationManager"
-         access-decision-manager-ref="accessDecisionManager">
-      <intercept-url pattern="/**" access="hasRole('ROLE_USER')"/>
-      <custom-filter position="PRE_AUTH_FILTER" ref="gaeFilter"/>
-      <csrf disabled="true" />
-   </http>
-
     <http use-expressions="true" entry-point-ref="defaultEntryPoint" authentication-manager-ref="authenticationManager"
```

Comment:
Maybe we can remove the defaultEntryPoint and use Spring's PropertyPlaceholderBlaBlabla... to configure the bean to use here, using as default value the auth0 bean (https://stackoverflow.com/questions/2513484/is-there-a-way-to-specify-a-default-property-value-in-spring-xml).

AFAIK, there is not PropertyPlaceholderBlaBlabla... yet in the Spring config, so it should be added if we opt for this.